cmd/uplink/main.go

```diff
@@ -91,18 +93,34 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
+	awsAccessCreds, awsSecretCreds, err := getGeneralAwsCreds()
+	if err != nil {
+		return err
+	}
+
 	o := map[string]interface{}{
 		"cert-path":       setupCfg.Identity.CertPath,
 		"key-path":        setupCfg.Identity.KeyPath,
 		"api-key":         setupCfg.APIKey,
 		"pointer-db-addr": setupCfg.SatelliteAddr,
 		"overlay-addr":    setupCfg.SatelliteAddr,
+		"access-key":      awsAccessCreds,
+		"secret-key":      awsSecretCreds,
 	}
 
 	return process.SaveConfig(runCmd.Flags(),
 		filepath.Join(setupCfg.BasePath, "config.yaml"), o)
 }
 
+func getGeneralAwsCreds() (string, string, error) {
```

Comment:
Somehow this method name confused me initially. I was wondering what are these "general" AWS credentials. This method actually generates random access and secret keys. "AWS" should be all caps in the name too.

So something like "generateAWSKeys" would look better.

Naming the return values would also make sense as the method returns two strings and its not clear which one is what just by looking at the method signature.

Perhaps something like:
```go
func generateAWSKeys() (accessKey string, secretKey string, err error)
```

Reply:
got it yeah; i'll make that change .